src/pkg/services/m365/api/sites.go

```diff
@@ -255,9 +253,26 @@ func (c Sites) GetDefaultDrive(
 // helpers
 // ---------------------------------------------------------------------------
 
-var ErrKnownSkippableCase = clues.New("case is known and skippable")
+const personalSitePath = "sharepoint.com/personal/"
 
-const PersonalSitePath = "sharepoint.com/personal/"
+func skippableSite(item models.Siteable) bool {
+	wURL := ptr.Val(item.GetWebUrl())
+	name := ptr.Val(item.GetDisplayName())
+
+	// personal (ie: oneDrive) sites have to be filtered out server-side.
+	if strings.Contains(wURL, personalSitePath) {
```

Comment:
Using `strings.Contains` makes me feel a bit uneasy. Should we parse out the URL and check against specific components. _I guess this is technically not a part of this PR, but thought I would mention._

Reply:
hmmm that's a good point. Let me make a ticket for that and then we can follow up on it. I'm not sure if there's other locations we're doing similar things

Reply:
filed #4555

Reply:
As long as we're poking at this, could one of y'all take another stab at getting the query filter to work?  Doing this server-side in any capacity is a smell.  I don't know what I was missing with the original query filter, but it *should* work (according to postman).  Just wasn't giving me anything in go.

Reply:
this was sort of the extent of the changes I was planning on making since it's mostly cleanup and there's other things I'm slated to work on right now. Could you please make a ticket about the query filtering if it doesn't already exist? That way we won't lose track of it

Reply:
no worries